packages/containers/src/List/List.connect.js

```diff
@@ -26,20 +21,26 @@ function getPagedItems(state, config) {
 
 export function mapStateToProps(state, ownProps, cmfProps) {
 	const props = {};
-	const config = {
+	let filteredItems;
+	const collectionItems = getCollectionItems(state, ownProps.collectionId);
+
+	props.config = {
 		collectionId: ownProps.collectionId,
-		items: ownProps.items,
+		items: collectionItems || ownProps.items || new List(),
 	};
 	if (ownProps.list) {
-		config.columns = ownProps.list.columns;
+		props.config.columns = ownProps.list.columns;
+	}
+	if (ownProps.defaultSaga !== false) {
```

Comment:
i'm not sure we want that, it mean that the saga would be loaded by default on the list ?
Why the props `defaultSaga` is checked to set a `saga` attribute then ? 
Shouldn't it test the `saga` attribute instead ?

Reply:
The saga attribute shouldn't be loaded when the defaultSaga flag is set to false, this enables the target application to load the custom saga (i.e., the default saga will not be called).
Existing applications that use ListContainer will function normally since the defaultSaga prop is newly introduced for server side list actions.

Reply:
But that's not how the saga mount feature works in CMF.
`props.data` should be a boolean to indicate if the attached saga should be mounted or not.

So if you don't want to mount the default saga, you just don't set the saga props in your cmf settings. No need to introduce a new props `defaultSaga`.

And I don't get the fact that you filter only if you want to use the default sagas. That sound weird.

Reply:
fixed, enabled saga under default props and disabled filter and sorting conditionally if the target app needs to override the default list operations